apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts

```diff
@@ -74,7 +74,7 @@ export const BILLING_BREAKDOWN_METRICS: Metric[] = [
   },
   {
     key: PricingMetric.REALTIME_PEAK_CONNECTIONS,
-    name: 'Realtime Peak Connections',
+    name: 'Realtime Peak Concurrent Connections',
```

Comment:
this one says "peak concurrent" and the one below says "concurrent peak", let's standardize on one